app/_locales/en/messages.json

```diff
@@ -5320,6 +5320,9 @@
   "stake": {
     "message": "Stake"
   },
+  "stateCorruptionDetectedErrorMessage": {
+    "message": "MetaMask's state file is damaged or missing. The files are possibly recoverable, so don't uninstall the MetaMask extension! You should contact MetaMask support."
```

Comment:
Could we offer users a simple guide or video for recovering corrupt storage on their own? Many users prefer solving issues without contacting support. If not, have we shared clear details about this error and recovery steps with the support team?

Reply:
I added a link to a support page that guides on how to do this in 2d978306ad